Mathlib/GroupTheory/SpecificGroups/Cyclic.lean

```diff
@@ -347,6 +347,28 @@ theorem IsCyclic.image_range_card (ha : ∀ x : α, x ∈ zpowers a) :
 #align is_cyclic.image_range_card IsCyclic.image_range_card
 #align is_add_cyclic.image_range_card IsAddCyclic.image_range_card
 
+@[to_additive]
+theorem IsCyclic.unique_zpow_zmod (ha : ∀ x : α, x ∈ zpowers a) (x : α) :
```

Comment:
LGTM, but maybe you can golf the proof using `zmodCyclicMulEquiv` (this results looks like a special case).

Thanks!

bors d+

Reply:
@riccardobrasca I can't find `​zmodCyclicMulEquiv` in the docs or with docs search or with moogle. Can you give me more clues?

Reply:
[zmodCyclicMulEquiv](https://leanprover-community.github.io/mathlib4_docs/Mathlib/GroupTheory/SpecificGroups/Cyclic.html#zmodCyclicMulEquiv) does work?

Reply:
?! I must have been on the mathlib3 docs? Sorry for wasting your time!

Reply:
This is weird!

Reply:
Getting back to the main point, I golfed my proof a little more and I can't see how to make it shorter with `zmodCyclicMulEquiv`; can you?

Reply:
I didn't have anything precise in mind, it only looks like a reasonable strategy. It's not very important.